app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if @user.errors.any? %>
```

Comment:
[IMO] We should not use instance variable inside partial, please use local variable